graph/index.html

```diff
@@ -65,5 +65,21 @@
 <script src="//cdn.rawgit.com/bassjobsen/Bootstrap-3-Typeahead/master/bootstrap3-typeahead.min.js"></script>
 <script src="main.js"></script>
 <script>main();</script>
+<!-- Piwik -->
```

Comment:
I anticipate: this whole file `graph/index.html` does not receive any environment configuration.

As a consequence it's not possible yet to execute this script conditionally.

But this is not an issue since Piwik ignores HTTP requests containing `localhost`.

We'll discuss at the next planning what to do with this file.